.vscode/settings.json

```diff
@@ -1,6 +1,6 @@
 {
     "python.testing.pytestArgs": [
-        "."
+        "tests"
```

Comment:
VSCode was not discovering suddenly the test for me. This solved it, but I have no idea why :D